modules/installation-generate-aws-user-infra-ignition.adoc

```diff
@@ -2,6 +2,7 @@
 //
```

Comment:
` modules/installation-generate-aws-user-infra-ignition.adoc` is no longer accurate now that you're handling GCP stuff in here too.

Reply:
@kalexand-rh I've renamed this module `installation-generate-ignition`, LMK if that's an issue.

Reply:
I take it `modules/modules/installation-generate-ignition-configs.adoc` is a different use case, going by the assembly list in the header.